src/Sylius/Bundle/ShopBundle/Resources/views/Product/Box/_content.html.twig

```diff
@@ -15,13 +15,11 @@
         <a href="{{ path('sylius_shop_product_show', {'slug': product.slug, '_locale': product.translation.locale}) }}" class="header sylius-product-name" {{ sylius_test_html_attribute('product-name', product.name) }}>{{ product.name }}</a>
 
         {% if not product.enabledVariants.empty() %}
-            {% set appliedPromotions = (product|sylius_resolve_variant).getAppliedPromotionsForChannel(sylius.channel) %}
             {% set price = money.calculatePrice(product|sylius_resolve_variant) %}
             {% set originalPrice = money.calculateOriginalPrice(product|sylius_resolve_variant) %}
+            {% set appliedPromotions = (product|sylius_resolve_variant).getChannelPricingForChannel(sylius.channel).getAppliedPromotions() %}
```

Comment:
Why did you change the order? And I guess that `getAppliedPromotionsForChannel` method still exists